sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/util/tracing/LoggingTracerProvider.java

```diff
@@ -186,5 +202,14 @@ public void end(Throwable throwable) {
                 }
             }
         }
+
+        /**
+         * Generates random id with given length up to 32 chars.
+         */
+        private static String getRandomId(int length) {
+            return CoreUtils.randomUuid().toString()
+                .replace("-", "")
+                .substring(32 - length);
```

Comment:
I think for clarity I'd make this just return the UUID without the `-` and have the caller handle doing the substring. Seeing `32 - length` is fairly confusing and likely could be error prone as you pass `15` for the length but you really get `17`.